src/notes/notes.controller.ts

```diff
@@ -0,0 +1,37 @@
+import { Controller, Get, Post,HttpCode, Param, Body, Patch, Delete } from '@nestjs/common';
+import { CreateNoteDto } from './dto/create-note.dto';
+import { UpdateNoteDto } from './dto/update-note.dto';
+import { NotesService } from './notes.service';
+
+@Controller('notes')
+export class NotesController {
+    constructor(private readonly notesService:NotesService){}
+
+    @Get()
+    getNotes() {
+        return this.notesService.findAll()
+    }
+
+    @Get(':id')
+    getNote(@Param('id') id:number){
```

Comment:
ogólnie nazwy nie potrzebują słowa "Note", ponieważ z racji, że jest to NotesController sama nazwa "get" sugeruje już co będzie typem zwracanym